source/jormungandr/jormungandr/street_network/kraken.py

```diff
@@ -99,9 +116,19 @@ def _direct_path(
 
             kraken_mode = 'car_no_park' if mode in (fm.taxi.name, fm.ridesharing.name) else mode
             direct_path_request['max_{mode}_duration_to_pt'.format(mode=kraken_mode)] = int(
-                direct_path_request['max_{mode}_direct_path_duration'.format(mode=mode)] / 2
+                request['max_{mode}_direct_path_duration'.format(mode=mode)] / 2
+            )
+            # if the crowfly distance between origin and destination is too large, there is no need to call kraken
+            crowfly_distance = crowfly_distance_between(
+                utils.get_pt_object_coord(pt_object_origin), utils.get_pt_object_coord(pt_object_destination)
             )
 
+            if (
+                crowfly_distance / float(direct_path_request['{mode}_speed'.format(mode=kraken_mode)])
```

Comment:
I suppose the devider value is never 0.

Reply:
i'll add a try catch!!

Reply:
@kadhikari  after a quick check, there is no need,  this param is always > 0 !  https://github.com/CanalTP/navitia/blob/dev/source/jormungandr/jormungandr/interfaces/v1/journey_common.py#L287